bee-message/src/payload/fpc/mod.rs

```diff
@@ -60,13 +89,19 @@ impl FpcPayloadBuilder {
     }
 
     /// Adds a collection of conflicts to the [`FpcPayloadBuilder`].
-    pub fn with_conflicts(mut self, conflicts: Conflicts) -> Self {
+    pub fn with_conflicts(
```

Comment:
I feel like this is a not really handy API for the user. Maybe we should take Vec here and try to convert them to the appropriate VecPrefix when calling `finish` and propagating errors ?